### ISSUE TITLE
not possible to build with gulp

### ISSUE THREAD

**Issue by mareksybilak** [picturepan2/spectre#549]
After `gulp build` I got 
```
assert.js:42
  throw new errors.AssertionError({
  ^

AssertionError [ERR_ASSERTION]: Task function must be specified
    at Gulp.set [as _setTask] (/Users/marek/Dev/test/spectre/node_modules/undertaker/lib/set-task.js:10:3)
    at Gulp.task (/Users/marek/Dev/test/spectre/node_modules/undertaker/lib/task.js:13:8)
    at Object.<anonymous> (/Users/marek/Dev/test/spectre/gulpfile.js:67:6)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
```

**Comment by mareksybilak** [comment]
gulpfile.js needs to be updated to work with gulp 4.0.0. 
Current version works well with gulp set to 3.9.1  in package.json 

"devDependencies": {
    "gulp": "3.9.1",
    "gulp-autoprefixer": "latest",
    "gulp-clean-css": "latest",
    "gulp-csscomb": "latest",
    "gulp-rename": "latest",
    "gulp-sass": "latest",
    "gulp-pug": "latest",
    "jstransformer-highlight": "latest",
    "jstransformer-markdown-it": "latest"
  },

**Comment by picturepan2** [comment]
@mareksybilak Yes, it has been fixed in v0.5.8 which will be released soon.
